crates/sui-bridge/src/tools/cli.rs

```diff
@@ -155,6 +167,85 @@ async fn main() -> anyhow::Result<()> {
 
             return Ok(());
         }
+        BridgeValidatorCommand::CommitteeRegistration {
```

Comment:
I'm not sure this file is the best place to add this command:
1. IMO we shouldn't ask for a `NodeConfig` which is a huge file but the only needed field is authority account key.
2. This file reminds me of `validator_commands.rs`: https://github.com/MystenLabs/sui/blob/main/crates/sui/src/validator_commands.rs#L221. This files contains some cli tools which validators can use for daily operations. The way it works is it loads rpc url and key as `WalletContext`. 


thoughts?

Reply:
ya this seem more suitable in validator commands, I can move it there.

the reason I load the node config is in our bridgenet setup, validator's account key can only be found in this file, and it's not in client.yml, I will see if we can change the deployment to add the account key to client.yaml.